src/test/java/core/basesyntax/service/operationhandler/SupplyHandlerTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.service.operationhandler;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class SupplyHandlerTest {
+    private SupplyHandler supplyHandler;
+
+    @BeforeEach
+    public void setUp() {
+        Storage.fruits.clear();
+        supplyHandler = new SupplyHandler();
+    }
+
+    @Test
+    public void handle_quantityNull_Ok() {
+        FruitTransaction item = new FruitTransaction(Operation.RETURN,
```

Comment:
why are you using `RETURN` in supply handler test? you have separate handlers in your code for each of these